src/test/java/co/cask/gcp/datastore/sink/DatastoreSinkConfigTest.java

```diff
@@ -0,0 +1,473 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.gcp.datastore.sink;
+
+import co.cask.cdap.api.data.schema.Schema;
+import co.cask.gcp.datastore.sink.util.DatastoreSinkConstants;
+import co.cask.gcp.datastore.sink.util.IndexStrategy;
+import co.cask.gcp.datastore.sink.util.SinkKeyType;
+import co.cask.gcp.datastore.util.DatastorePropertyUtil;
+import com.google.cloud.datastore.PathElement;
+import com.google.datastore.v1.client.DatastoreHelper;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
nit: we usually only import classes, not static method imports. So other code will do Assert.assertEquals() while importing only org.junit.Assert